.gitignore

```diff
@@ -50,3 +50,6 @@ docs/tikz/*.png
 
 # Exclude our demo/test tmux directory
 tmux/
+
+# Exclude CLion directory
+.idea/
```

Comment:
I think personal workflow and tooling exclude list should not make it to the shared repository. It's good that we all use our favorite tool, the shared repository should only deal with what our build process produces. Please consider moving that to `.git/info/exclude` instead.